src/Elements/qualsource.cpp

```diff
@@ -54,7 +54,7 @@ void QualSource::setStrength(Node* node)
 {
     strength = base;
     if ( pattern ) strength *= pattern->currentFactor();
-    if ( type == MASS ) strength *= 60.0;         // mass/min -> mass/sec
+    if ( type == MASS ) strength /= 60.0;         // mass/min -> mass/sec
```

Comment:
converting mass flow rate from min. to sec.

Reply:
This change is correct - thank you.